### PR TITLE
Add instructions "How to create translation files when using Create React App"

### LINKED ISSUE #869: How to create translation files when using Create React App
is it possible to use [babel-plugin-react-intl](https://github.com/yahoo/babel-plugin-react-intl) when using [create-react-app](https://github.com/facebookincubator/create-react-app)?

if not, any other way to generate the relevant files?

https://github.com/yahoo/react-intl/pull/669
https://github.com/yahoo/react-intl/pull/673

### PR DESCRIPTION
**Pull request by mulyoved**
resolve #869

---
This change is [Reviewable](https://reviewable.io/reviews/yahoo/react-intl/916)

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

File: examples/hello-world/package.json [hunk 1]
```diff
@@ -16,6 +21,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build"
+    "build": "react-scripts build",
+    "translate": "cross-env NODE_ENV=production babel ./src > NUL"
```
Comment (okuryu) [inline examples/hello-world/package.json]: what's `NUL`?

File: examples/hello-world/translations/messages/src/App.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "id": "welcome",
+    "defaultMessage": "Hello {name}, you have {unreadCount, number} {unreadCount, plural, one {message} other {messages}}"
+  }
+]
```
Comment (okuryu) [inline examples/hello-world/translations/messages/src/App.json]: missing a newline.

File: examples/hello-world/package.json (hunk 1 above)
Comment (mulyoved) [inline examples/hello-world/package.json]: cross-platform way to discard the output

File: examples/hello-world/translations/messages/src/App.json (hunk 2 above)
Comment (mulyoved) [inline examples/hello-world/translations/messages/src/App.json]: added

File: examples/hello-world/package.json (hunk 1 above)
Comment (okuryu) [inline examples/hello-world/package.json]: This doesn't works on macOS.
```
$ npm run translate

> react-intl-example-hello-world@1.0.0 translate /Users/okuryu/work/react-intl/examples/hello-world
> cross-env NODE_ENV=production babel ./src > NUL

$ ls -l
total 24
-rw-r--r--    1 okuryu  staff   3682  5 18 22:40 NUL
-rw-r--r--    1 okuryu  staff    997  5 16 23:48 README.md
drwxr-xr-x  766 okuryu  staff  26044  5 16 23:50 node_modules
-rw-r--r--    1 okuryu  staff    717  5 18 22:40 package.json
drwxr-xr-x    3 okuryu  staff    102  5  3 21:29 public
drwxr-xr-x    4 okuryu  staff    136  5 16 23:53 src
drwxr-xr-x    3 okuryu  staff    102  5 16 23:48 translations
```

Comment (mulyoved) [inline examples/hello-world/package.json]: I see you have the translations folder, check you have examples/hello-world/translations/messages/src/App.json

**Comment by wguttmn** [comment]
Any chances to get this reviewed and merged ?

**Comment by kbiedrzycki** [comment]
How to use other languages?

File: examples/hello-world/package.json (hunk 1 above)
Comment (okuryu) [inline examples/hello-world/package.json]: Sorry for delay! I meant the created `NUL` file shouldn't be. I prefer non-redirecting to the `NUL` like this:
```
    "translate": "cross-env NODE_ENV=production babel ./src"
```

**Comment by alanrubin** [comment]
I strongly recommend not using the babel plugin for that, I have tried that road and you don't get a clear way on how to manage the translations back and forth. I recommend using https://github.com/evenchange4/react-intl-cra (see a sample translation flow at https://github.com/evenchange4/react-intl-po-example), you don't need any configuration change to babel to use that, only adding a few lines to package.json.

**Comment by mulyoved** [comment]
@alanrubin thanks for the link, 

I close this PR I agree it is not a good idea if we have option that doesn't need any configuration change to babel

**Comment by alanrubin** [comment]
@mulyoved As create-react-app is very popular, maybe it is worth having an entry in the README or Wiki mentioning the links I have described... What do you think ?

**Comment by mulyoved** [comment]
Sure this is a good idea, I am sure PR will be accepted :)

**Pull request closed without merging**